server/components/fill_teams_test.js

```diff
@@ -0,0 +1,46 @@
+/**
+* Created by Sammy on 1/13/15
+**/
+
+var db = require('../api/models');
+
+var Fill = (function() {
+  var stockTeams = [
+    {name: "Albany Alphas"},
+    {name: "Alaska Arctics"},
+    {name: "Baltimore Spirits"}
+  ]
+
+  var stockPlayers = [
+    {name: "Rodger Umaechi",
+    realWorldTeam: "ALB"},
+    {name: "Apiatan Redmane",
+    realWorldTeam: "ALB"},
+    {name: "Carl Esteban",
+    realWorldTeam: "ALK"},
+    {name: "Ambrose Slade",
+    realWorldTeam: "ALK"},
+    {name: "Silvia Windcreek",
+    realWorldTeam: "BAL"},
+    {name: "Dirk von Stryker",
+    realWorldTeam: "BAL"}
+  ]
+
+  var _fillTeams = function(user) {
+    if (user.role !== 'admin') {
+      console.log("Error, user is not an admin, user role is: " + user.role);
+      return false;
+    }
+    return db.Team.bulkCreate(stockTeams)
+  }.then(function() {
```

Comment:
I seem to be hitting an error here, saying that there is no function 'then'. Not sure what's causing it - or why. I'll probably fix it soon, but wondering if I can get some insight on the reasoning. It might just be a simple syntax error (I remember I wasn't sure on the curly braces being able to follow with a method when I wrote this.)